### ISSUE TITLE
Screenshot Button does not work on Yume

### ISSUE THREAD

**Issue by Romeo923** [qxb3/dotfiles#6]
In `.config/ags/windows/bar/Bar.js` recommend replacing
```javascript
const ScreenShotButton = Widget.Button({
    className: 'screenshot_button',
    child: Widget.Label('󰄄'),
    onPrimaryClick: () => Utils.subprocess(
      ['bash', '-c', '/home/qxb3/.config/hypr/scripts/screenshot.sh p'],
      (output) => print(output)
    )
  })
```
with
```javascript
const ScreenShotButton = Widget.Button({
    className: 'screenshot_button',
    child: Widget.Label('󰄄'),
    onPrimaryClick: () => Utils.subprocess(
      ['bash', '-c', '/home/$USER/.config/hypr/scripts/screenshot.sh p'],
      (output) => print(output)
    )
  })
```

**Comment by qxb3** [comment]
Hi! Wont `~/` work fine too?

**Comment by Romeo923** [comment]
Yep, `~/` should work too!

**Comment by qxb3** [comment]
fixed [624b824](https://github.com/qxb3/gruvbox.hypr/commit/624b824d35762a4a2388be83f7c707be5e4c0fd7)
